low-level/sensors/impl/generic/node_data.py

```diff
@@ -250,27 +252,14 @@ def _fetch_nw_status(self):
     def fetch_nw_cable_conn_status(self, interface):
         carrier_status = None
         try:
-            carrier_status = self._utility_instance.fetch_nw_cable_status(self.nw_interface_path, interface)
+            carrier_status = Network().get_link_state(interface)
+        except NetworkError as err:
+            logger.debug(err)

```

Comment:
this should be logger.error only

Reply:
That might not be the error per se, 
if some interface is disabled, then it won't have a carrier file so the Exception will be NetowrkError since OS related errors are handled by NetworkError, In this case we don't want to keep logging error every iteration, this is the expected UNKNOWN state. That's why I logged in debug. If the Exception is unknown or unexpected then only I am logging it.

Reply:
Add these details in code then, why such info is not shared until asked ? These are not obvious things we can relate just by looking at the code being changed or even if someone looking at just this or limited code

Reply:
Added in the comments please check.

Reply:
Its not crisp, but thats fine.